frontend/packages/ux-editor/src/components/Elements/LayoutSetsContainer.test.tsx

```diff
@@ -51,22 +57,6 @@ describe('LayoutSetsContainer', () => {
     expect(appContextMock.onLayoutSetNameChange).toHaveBeenCalledWith('test-layout-set-2');
   });
 
-  it('should render add and delete subform buttons when feature is enabled', () => {
```

Comment:
That's right, 🙂  I updated now the both tests to match display /not display `Slett underskjema` button and removed the part related to `Opprett underskjema` button because we no longer have this button.